apps/src/lib/node/ledger/tendermint_node.rs

```diff
@@ -454,6 +454,14 @@ async fn write_tm_genesis(
     genesis.genesis_time = genesis_time
         .try_into()
         .expect("Couldn't convert DateTimeUtc to Tendermint Time");
+    genesis.consensus_params.block = Some(block::Size {
+        // maximum size of a serialized Tendermint block
+        // cannot go over 100 MiB
+        max_bytes: 100 << 20,
+        // gas is metered app-side, so we disable it
```

Comment:
would make this comment a TODO and link it to a GH issue on metering gas app side

Reply:
I think we're already metering gas app-side
https://github.com/anoma/namada/blob/926f47bf80d86a12c5ba2b044c113a31bc77deaf/apps/src/lib/node/ledger/shell/prepare_proposal.rs#L45-L52

Reply:
oops yeah, I got this mixed up with setting a max gas cap on blocks (coming after the block allocator I guess)

Reply:
yeah, that's still a TODO